_includes/layouts/base.njk

```diff
@@ -46,7 +46,42 @@
     <link rel="preconnect" href="/" crossorigin>
     <script async defer src="{{ "/js/min.js" | addHash }}"></script>
     {% if googleanalytics %}
-    <script async defer src="{{ "/js/cached.js" | addHash }}"></script>
+      <script async defer src="{{ "/js/cached.js" | addHash }}"></script>
```

Comment:
With this being set to async, I don't think it can be guaranteed that `ga` function has been defined in time for the inline script below.

Reply:
That's a great point, I didn't consider this. I could:

1. Remove async (and keep defer). I believe this will execute `cached.js` which contains `ga`  before the inline script. 
2. Add defer to the inline script so both are executed after HTML is parsed.

Do you have any concerns removing async here? I think the effect would be that `cached.js` gets executed only after parsing is complete.

Reply:
I'd really like to keep the async here (Not block `DOMContentLoaded` on the script).

Let's switch this to use dynamic script injection inside of `main.js` to load the web-vitals.js script.

Reply:
Ok, I have got this working locally.  What would be the best way to capture opt-in from `sendWebVitals` within `metadata.json` ?